CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.5.0 (Unreleased)
```

Comment:
In your opinion, is this ready to be tagged and released, or merge for now an perform some more testing?

Also, did you try sending/receiving `[]map[any]any` to RabbitMQ (and/or a different broker) to ensure this is working as expected (I'm guessing you've tried this for the scenario outlined in the linked issue)?

Reply:
Thanks for confirming.

If you want, please feel free to set the release date in the changelog to today's date and I'll tag the release once this is merged.

Reply:
Thanks for the contribution!

Once CI is done I'll merge and tag.